src/main/kotlin/com/hypto/iam/server/service/OrganizationsService.kt

```diff
@@ -43,43 +45,49 @@ class OrganizationsServiceImpl : KoinComponent, OrganizationsService {
         val organizationId = idGenerator.organizationId()
 
         val identityGroup = identityProvider.createIdentityGroup(organizationId)
```

Comment:
We need to add a logic to delete the identityGroup and user from cognito when the transaction fails.